tests/k8st/infra/calico-kdd.yaml

```diff
@@ -661,6 +661,8 @@ spec:
             # Enable IPIP
             - name: CALICO_IPV4POOL_IPIP
               value: "Always"
+            - name: FELIX_SERVICELOOPPREVENTION
```

Comment:
This feature broke the STs. Disabling for now to get them working, but we need to re-enable this before release.